include/io.h

```diff
@@ -28,6 +28,15 @@ class IO {
   //! Return the number of points per coordinate
   unsigned ngauss_points() const { return ngauss_points_; }
 
+  //! Return the element type
```

Comment:
Why should an `io` class return `element_type`, `nvertices`?

Reply:
Because the element type as previously discussed does make different vertices (nodes)

Reply:
`IO` class shouldn't worry about `element_type`, `nvertices` etc. They are the problem of the mesh. My issue is with abstraction, who owns what. Why should IO have anything to do with what `element_type` the input has, it should be the problem of the mesh.